smartsim/_core/utils/helpers.py

```diff
@@ -140,6 +140,8 @@ def expand_exe_path(exe: str) -> str:
     :raises FileNotFoundError: if executable cannot be found
     """
 
+    if not exe:
+        raise ValueError("No executable provided")
```

Comment:
Does this ``ValueError`` need to be called out in the doc strings?

Reply:
nope. I can change it back